CRM/Custom/Import/Form/MapField.php

```diff
@@ -6,7 +6,7 @@
 class CRM_Custom_Import_Form_MapField extends CRM_Contact_Import_Form_MapField {
   protected $_parser = 'CRM_Custom_Import_Parser_Api';
   protected $_mappingType = 'Import Multi value custom data';
-  protected $_highlightedFields = array();
+//  protected $_highlightedFields = array();
```

Comment:
Not sure why this is commented out 

I would have thought later down you could have done $this->_highlightedfields[] = ... and left the page assignment to $this->_highlightedfields.